lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -169,17 +170,22 @@ class DocumentImpl extends NodeImpl {
       actAsIfLocationReloadCalled: () => this._location.reload()
     });
 
-    this._workingNodeIterators = [];
-    this._workingNodeIteratorsMax = privateData.options.concurrentNodeIterators === undefined ?
-                                    10 :
-                                    Number(privateData.options.concurrentNodeIterators);
-
-    if (isNaN(this._workingNodeIteratorsMax)) {
-      throw new TypeError("The 'concurrentNodeIterators' option must be a Number");
-    }
+    if (hasWeakRefs) {
+      this._workingNodeIterators = new IterableWeakSet();
+    } else {
+      this._workingNodeIterators = [];
+      this._workingNodeIteratorsMax =
```

Comment:
It looks like we never ported over the concurrentNodeIterators option to the "new" API: https://github.com/jsdom/jsdom/blob/69b0205ed4cdafe94f9ffea519e00dcc67e17f1f/lib/api.js#L285 . Since nobody has noticed, let's hard-code it to 10, and remove the validation code?